src/components/OpportunityCard/OpportunityCard.js

```diff
@@ -197,17 +196,72 @@ export default function OpportunityCard(props) {
             daoResult.name ? setCommunityName(daoResult.name) : setCommunityName('')
             daoResult.logo ? setLogo(daoResult.logo) : setLogo(defaultImage)
           }
+          
+         
         }
-        
-        let mounted = true
-        if(mounted){
-          fetchData()
-              .then((res) => {
-                
-              })
-        return () => mounted = false
```

Comment:
put this back in where you call fetchData - need to wrap fetchData with the mounted lines to get rid of the react memory leak error we see pop up